PocketKit/Sources/Textile/Views/LottieRefreshControl.swift

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Modified from https://stackoverflow.com/questions/53123171/custom-uirefreshcontrol-with-lottie
+
+import UIKit
+import Lottie
+
+public class LottieRefreshControl: UIRefreshControl {
+    fileprivate let animationView: LottieAnimationView
+    fileprivate var isAnimating = false
```

Comment:
curious could we make these `private` instead?

Reply:
@cyndichin copy pasted from the original link in the header, i could if we want.

Reply:
ah okay thanks np! just prefer to be on the side of most restricted if possible

is this ready for review again?